src/App.js

```diff
@@ -1,16 +1,36 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog.js'
 
 const App = () => {
+  const [entries, setEntries] = React.useState(chatMessages);
```

Comment:
👍 We need a piece of state to track changes to our messages, initialized from the json file we loaded.